bugbear.py

```diff
@@ -4,6 +4,7 @@
 import logging
 import re
 import sys
+import typing as t
```

Comment:
Wasn't sure whether to use ``from __future__ import annotations`` and use ``nodes: list`` at line 442, or to use this form, so picked this form.

Reply:
We can't support that due to 3.6 support I think.
```suggestion
from typing import List
```

I really dislike `import typing as t`

`def hard_to_read_typing(x: t.List[t.Dict[t.Union[str, bool]]]) -> t.Dict[t.List[t.Any]]:`

is terrible to read imo compared to:

`def readable_typing(x: List[Dict[Union[str, bool]]]) -> Dict[List[Any]]:`

Reply:
I had thought that there might be a version issue. At least we can use typing.